kafka-on-docker/README.md

```diff
@@ -0,0 +1,145 @@
+<!-- title: How to run Kafka locally on Docker -->
+<!-- description: In this tutorial, learn how to run Kafka broker locally on your laptop, with step-by-step instructions and supporting code. -->
+
+# How to run a Kafka broker locally with Docker
+
+When building or testing Apache®Kafka applications, it can be very helpful to have a local Kafka broker running locally.  Using [Docker](https://www.docker.com/) is possibly the fastest way to get a locally running Kafka broker
```

Comment:
```suggestion
When building or testing Apache Kafka® applications, it can be very helpful to have a Kafka broker running locally.  Using [Docker](https://www.docker.com/) is possibly the fastest way to get a locally running Kafka broker.
```